packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -124,13 +139,14 @@ static inline BOOL IsHorizontalNavigationWithinZoneAction(FocusZoneAction action
 
 - (BOOL)acceptsFirstResponder
 {
-	// Accept firstResponder on FocusZone itself in order to reassign it to defaultTabbableElement.
+	// Accept firstResponder on FocusZone itself in order to reassign it within the FocusZone.
 	return !_disabled;
 }
 
 - (BOOL)becomeFirstResponder
 {
-	return !_disabled && [[self window] makeFirstResponder:_defaultTabbableElement ?: [self nextValidKeyView]];
+	NSView *keyView = _defaultKeyView ?: GetFirstKeyViewWithin(self);
```

Comment:
where do we actually set this property?

Reply:
RCTFocusZoneManager.m, line 43. See my change in that file.

Reply:
ah thanks! so... if nothing is to put focus on within itself, it will still put the firstreponder to the next one(sibling view)?

Reply:
Yeah it defaultKeyViw is nil (as is the case with Contextualmenu), then it uses my helper function to focus on the first key view within itself.